StreemKit/0.30.0-swift55/StreemKit.podspec.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "StreemKit",
+  "version": "0.30.0-swift55",
+  "summary": "Streem IOS SDK",
+  "description": "The Streem SDK supports on-demand video calling between professionals and consumers.\nAR Tools enhance the collaboration, and using image analysis, can automatically\nextract model/serial numbers from products, and save important project details.",
+  "homepage": "https://streem.pro",
+  "platforms": {
+    "ios": "13.0"
```

Comment:
Just like the Streemkernel PR, I don't understand why the platform change to `14` isn't reflected here.

Reply:
I thought we only did the `-swift55` thing for GuidedScan. This is for 0.30.0 of StreemKit, right? Minimum version should be 13.2.1, since that's what we build with.

Reply:
This one is supposed to be for a release of StreemApp-Lowes but I'm wondering if we even need it anymore since we could use GuidedScanNow. So I'll have to clarify that.